homeassistant/components/light/osramlightify.py

```diff
@@ -198,6 +196,8 @@ def turn_on(self, **kwargs):
             self._luminary.set_luminance(
                 int(self._brightness / 2.55),
                 transition)
+        else:
+            self._luminary.set_onoff(1)
```

Comment:
Lights can also be turned on by automations, or you can simply call the `light.turn_on` service from the Developer Tools in the UI, with Service Data like this: `{"color_name":"red"}`